### DISCUSSION TITLE
posts/easy-action/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#31] [Comments]
# posts/easy-action/

启动一个化学反应需要足够的活化能，催化剂可以降低化学反应所需的活化能，使得化学反应更容易发生。日常生活中一些小习惯也有催化剂的效果，让行动更丝滑，我称它们为「行动催化剂」。
[&hellip;] 近两个月，我体验了一款「行动催化剂」：不关电脑和软件。
[&hellip;] 俺的日常工作、学习和生活，除了运动外的大部分时间都坐在电脑前。工作以来，俺的电脑快达成连续一个月的开机时间，这对工作前的我来说 &hellip;

https://shitao5.org/posts/easy-action/

**Comment by yihui** [comment]
关于关电脑，我以前也是每天都关机。后来变成从来不关机（只休眠）的原因是感觉每天事情都没做完、第二天打开电脑要接着做。近两年我有点想回到关机的时代，就是觉得电脑里一大坨永远都做不完、看不完的事情让我有点心累。

关于封面图片，我其实早就想吐槽当代网页的这个毛病了，就是凡是写文章一定要配个封图，倒不是说封图本身相不相关（这是次要的），主要是找图太特么费事了。本来是个可选的锦上添花的事情，到最后搞得像强迫症一样一定要花时间走这么个仪式。刚开始的时候写三篇五篇文章，觉得封图好好看、让人好有动力写下去，而写了百十篇后，就会觉得配图简直是个时间黑洞，吸走了大半写作的能量。写作的流程就应该十分简单粗暴：要写，就从写字开始，而不是第一步先找图。

其实这个 hugo-paged 主题也一样有个小黑洞，就是标题后面的那个图标。找一个理想的图标字符所花的时间也不少。

作为轻度收纳强迫症患者，我提一点建议，就是你可以考虑把评论从 <https://github.com/Shitao5/blog-comments/discussions> 迁移到 <https://github.com/Shitao5/shitao-blog/discussions>（后者的讨论区启用后，在前者的每个讨论页面上点“Transfer this discussion”）。能归置到一块儿的东西就归置到一块儿，[聚合才有力量](https://yihui.org/cn/2019/03/influence-focus/)。评论另立一个库存放，这在技术上没啥道理。

好了，上面都不是重点，重点在于你这次搬家时迁移了文章的目录，所以文章的 URL 都变了。若要做一个万年长存的网站，就尽量别更改网页的链接；实在要改，就一定要[做重定向](https://yihui.org/en/2017/11/301-redirect/)（可以用通配符），否则过去分享出去的链接都成了死链接，比如：https://github.com/Shitao5/blog-comments/discussions/29

同样，因为文章链接变化，导致 Giscus 的评论映射失效，你需要修改每一项讨论页面的标题、换成新链接。还好你的评论数量不多，可以手动改一下，否则可以程序批量修改。这件事最好尽快做，免得有人在旧日志下创建了新评论、到时候两个讨论页面无法合并。

**Reply by Shitao5** [reply]
> 好了，上面都不是重点，重点在于你这次搬家时迁移了文章的目录，所以文章的 URL 都变了。

读到重点心拔凉拔凉的（拍大腿）。每次捣鼓博客都手忙脚乱，顾头不顾尾的，谢谢 yihui 提醒和提供方法，俺抓紧时间修复下！

评论分库是之前用 utterances 的遗产，当时还没有 Github Discussions 的功能，评论会放置在 Github Issue 中。分库是为了让评论和真正的 Issue 分开，当时也是看了一篇什么技术贴搞的。现在切换到 giscus，并且有了 Github Discussions 功能，确实可以使用「影响力聚焦」原则，把评论和博客合到一个库里了。船小好调头，这次俺一并修改掉。

> 近两年我有点想回到关机的时代，就是觉得电脑里一大坨永远都做不完、看不完的事情让我有点心累。

就我目前简短的经历来讲，关机不大会解决心累的问题。心累主要是这坨做不完、看不完的事情直接压在心上了，作用逻辑是：`事务 -> 人`；电脑开着或者使用一些笔记软件记录的逻辑是：`事务 -> 电脑/笔记 -> 人`。关键是要从 `电脑/笔记 -> 人` 这一环中解放出来，个人的理想状态就是离开电脑/笔记就可以抛开这些事务，再次回到电脑/笔记时又可以快速进入状态（这是支持不关机的理由哈哈）。感觉有点没心没肺，或许我该庆幸还没有体会到你这么忙碌的阶段。

> 写了百十篇后，就会觉得配图简直是个时间黑洞，吸走了大半写作的能量。

现实案例：诗涛本人以身试法，写了十几篇就不行了，吸走了全部写作的能量。

> 写作的流程就应该十分简单粗暴：要写，就从写字开始，而不是第一步先找图。

道理如此，但每次新建完博客，看着那 YAML 里那句 `img: ~`，就按捺不住找图的心，姑且称它为「封面图的诱惑」吧！这背后应该是我的写作功夫还没上道，潜意识觉得找图比写作舒服，所以跑去找图了。还是换个主题断了封面图的念想，不然它老对我念念不忘哈哈！

> 其实这个 hugo-paged 主题也一样有个小黑洞，就是标题后面的那个图标。找一个理想的图标字符所花的时间也不少。

不能对找图太吝啬呀哈哈，这个图标一劳永逸，应该算在建站的时间里边，不算在写博客的时间。

**Reply by yihui** [reply]
拔凉拔凉是正常的第一反应，不过你大可放心，我跟你讲的话绝对不会有什么会真的需要你拔凉拔凉的东西，不用紧张。我说是重点，其实也未必有多重要，尤其是你的网站规模小，流量可能也不大，所以就算几天没有重定向，也不会影响到几个人。我只是让你在初期就先养成这样一个习惯。

Utterances 时代其实也可以通过指定标签来将普通事项与评论创建的事项分开的，就像现在 Giscus 也会指定一个评论专属的分类，与其它讨论区别开来。

我越来越觉得我可以发展一个新职业，就是网站整理魔法师，帮人把网站收拾整齐，就像《怦然心动的整理魔法》一样，近藤麻理惠女士帮人整理房间，而我可以整理虚拟物品，哈哈。

**Reply by Shitao5** [reply]
嗯嗯，正在总结博客改版中的注意事项。`修改文章目录 -> 重点向 + 评论映射`  这样的意识之前是没有的嘞，努力养成习惯，感谢 yihui 提醒嘿嘿！

> Utterances 时代其实也可以通过指定标签来将普通事项与评论创建的事项分开的，就像现在 Giscus 也会指定一个评论专属的分类，与其它讨论区别开来。

之前走弯路了哈哈，现在走上正轨。

> 我越来越觉得我可以发展一个新职业，就是网站整理魔法师，帮人把网站收拾整齐，就像《怦然心动的整理魔法》一样，近藤麻理惠女士帮人整理房间，而我可以整理虚拟物品，哈哈。

我身边有朋友请人整理过房间，效果一般但价格不菲。收拾整齐不仅需要收拾技术，还需要有美感，对于收拾实物来说甚至后者更重要。网站整理魔术师太酷啦，blogdown 把一部分魔术技术化了，但万事万物总倾向于混乱（就像俺的 `content/` 目录都整理了好多回了），这时若有魔法师或许少走几年弯路呢！

**Reply by CyrusYip** [reply]
> 读到重点心拔凉拔凉的（拍大腿）。每次捣鼓博客都手忙脚乱，顾头不顾尾的，谢谢 yihui 提醒和提供方法，俺抓紧时间修复下！

如果还有下次，可以先把评论功能关了，这样读者就不能新建评论了。

推荐一下睡眠，可以省电而且唤醒很快。我自己比较喜欢休眠，因为它是断电的。看了你这篇文章，突然发现我的开机输入三次密码（BIOS、硬盘、系统）的过程简直是行动抑制剂。开机太痛苦了，怪不得我只有睡前才休眠电脑。

**Reply by Shitao5** [reply]
> 如果还有下次，可以先把评论功能关了，这样读者就不能新建评论了。

俺争取没有下次哈哈……那天早上看到 yihui 的评论俺就从床上翻下来开始捣鼓，好在评论数量不多，硬着头皮在去打工前改完了。其实中途我也想过，可以短暂关闭评论功能，等搬完就评论再打开，但想到从用户角度看（虽然用户寥寥无几），这样的产品显得不够友好。

> 突然发现我的开机输入三次密码（BIOS、硬盘、系统）的过程简直是行动抑制剂。

这安全等级也太高了，果然安全和便捷之间有取舍。我现在几乎没有安全的需求，要是有人可以摸到我电脑，那俺家应该也被连窝端了哈哈。

睡眠和休眠好像都会断开网络，可能是我之前用校园网的原因。只关屏幕真得太顺畅到有一种电脑在值班的感觉，哪怕是几分钟的空闲时间，也可以敲两下键盘把屏幕点亮，看几行文字。
